modules/channel-web/src/backend/db.ts

```diff
@@ -91,27 +144,19 @@ export default class WebchatDb {
       message_raw: this.knex.json.set(raw),
       message_data: this.knex.json.set(data),
       payload: this.knex.json.set(payload),
-      sent_on: this.knex.date.now()
+      sent_on: new Date().toISOString()
```

Comment:
Date is handled differently depending on postgres or sqlite, please keep knex.date.now. It's out implementation

Reply:
The problem I had with knex.data.now() is that the date is computed when the query is run. This meant that messages batched together would all have the same date and be out of order

Reply:
Finally I ended up using date.format()